src/test/java/hudson/scm/SVNWorkingCopyTest.java

```diff
@@ -23,57 +23,56 @@
  */
 package hudson.scm;
 
-import hudson.model.AbstractProject;
-import hudson.model.FreeStyleBuild;
 import hudson.model.FreeStyleProject;
-import hudson.model.queue.QueueTaskFuture;
-import hudson.scm.subversion.WorkspaceUpdater;
-import hudson.scm.subversion.WorkspaceUpdaterDescriptor;
+import java.io.File;
+import static org.junit.Assert.*;
+import org.junit.Ignore;
+import org.junit.Test;
 import org.jvnet.hudson.test.Issue;
-import org.tmatesoft.svn.core.internal.wc.SVNStatusUtil;
-import org.tmatesoft.svn.core.internal.wc.admin.*;
+import org.tmatesoft.svn.core.internal.wc.admin.SVNAdminArea14;
+import org.tmatesoft.svn.core.internal.wc.admin.SVNAdminArea15;
+import org.tmatesoft.svn.core.internal.wc.admin.SVNAdminArea16;
 import org.tmatesoft.svn.core.internal.wc17.db.ISVNWCDb;
-import org.tmatesoft.svn.core.io.SVNRepositoryFactory;
 import org.tmatesoft.svn.core.wc.SVNClientManager;
-import org.tmatesoft.svn.core.wc.SVNStatus;
-import org.tmatesoft.svn.core.wc.SVNStatusClient;
-import org.tmatesoft.svn.core.wc2.SvnGetStatus;
-
-import java.io.File;
-import org.junit.Ignore;
 
 /**
  * @author schristou88
  */
 @Ignore("TODO org.tmatesoft.svn.core.SVNException: svn: E175002: PROPFIND of '/trunk/hudson/test-projects/trivial-ant': 405 Method Not Allowed (https://svn.jenkins-ci.org)")
```

Comment:
We do: `svn.jenkins-ci.org` is long dead. Another case where we need to use `docker-fixtures` to set up a test server.

(There is another test floating around here that connects to `subversion.tigris.org`, which still works, but is poor style.)